pkg/spec/cluster.go

```diff
@@ -30,7 +30,7 @@ import (
 // TODO: supports object store like s3
 type StorageType string
```

Comment:
Please remove the above. I'm seeing a lot of artifacts that I find it weird they're still around. Seems my "grep" skills are too rusty.